06-getting-your-class-going-with-jupyter.md

```diff
@@ -324,6 +324,15 @@ which provide free trials or a "freemium" pricing model. They include:
     like a way to create a course, invite students, and deploy
     auto-graded assignments.
 
+* HubHero ([https://hubhero.net](https://hubhero.net)) provides dependable
```

Comment:
Since we, the book authors, haven't tried the service, we should avoid assessments like "dependable."